inc/projecttask.class.php

```diff
@@ -1299,11 +1299,22 @@ static function populatePlanning($options=array()) {
                      AND ";
       }
 
+      $DONE_EVENTS = '';
+      if (!$options['display_done_events']) {
+         $DONE_EVENTS = "`glpi_projecttasks`.`percent_done` < 100
+                         AND (glpi_projectstates.is_finished = 0
+                              OR glpi_projectstates.is_finished IS NULL)
+                         AND ";
+      }
+
       $query = "SELECT `glpi_projecttasks`.*
                 FROM `glpi_projecttaskteams`
                 INNER JOIN `glpi_projecttasks`
                   ON (`glpi_projecttasks`.`id` = `glpi_projecttaskteams`.`projecttasks_id`)
+                LEFT JOIN `glpi_projectstates`
```

Comment:
i think the difference must be very small with and without join (a microsecond ?)
And in fact, ``$options['display_done_events'] = false`` is the default behavior.

I kept this options (maybe plugins use it) but on glpi side, i think it's never used (and so join always required)